test/integration/failing-multiple-groups/Foo.hs

```diff
@@ -0,0 +1,22 @@
+module Foo where
```

Comment:
I think ideally you want to be able to guess from the path to what feature a specific test file relates to.  For that reason I think putting this under `test/ integration/fail-fast/`